[Bug]: kured_version compare - fatal error

### Description

Omitting `kured_version` from my `kube.tf` causes the following error:

```

│ Error: Invalid function argument
│
│   on .terraform/modules/kubernetes.kube-hetzner/locals.tf line 19, in locals:
│   19:   kured_yaml_suffix = provider::semvers::compare(local.kured_version, "1.20.0") >= 0 ? "combined" : "dockerhub"
│     ├────────────────
│     │ while calling provider::semvers::compare(version_a, version_b)
│     │ local.kured_version is null
│
│ Invalid value for "version_a" parameter: argument must not be null.
╵
make: *** [plan] Error 1
```

Note: Manually setting `kured_version = "1.20.0"` as an input variable resolves this.

### Kube.tf file

```terraform
module "kube-hetzner" {
  source  = "kube-hetzner/kube-hetzner/hcloud"
  version = "2.18.4"

  hcloud_token = var.hcloud_token

  ssh_public_key  = "foo"
  ssh_private_key = "bar"

  ssh_additional_public_keys = []

  network_region      = "eu-central"
  existing_network_id = [var.network_id]
  network_ipv4_cidr   = var.network_cidr

  control_plane_nodepools = [
    {
      count        = var.control_plane_nodepool.count
      name         = var.control_plane_nodepool.name
      server_type  = var.control_plane_nodepool.server_type
      location     = var.control_plane_nodepool.location
      labels       = var.control_plane_nodepool.labels
      taints       = var.control_plane_nodepool.taints
      swap_size    = var.control_plane_nodepool.swap_size
      zram_size    = var.control_plane_nodepool.zram_size
      selinux      = false
      disable_ipv4 = true
      disable_ipv6 = true
      network_id   = var.network_id
      backups      = var.control_plane_nodepool.backups
    }
  ]

  agent_nodepools = [for np in var.agent_nodepools : {
    count        = np.count
    name         = np.name
    server_type  = np.server_type
    location     = np.location
    labels       = np.labels
    taints       = np.taints
    swap_size    = np.swap_size
    zram_size    = np.zram_size
    selinux      = false
    disable_ipv4 = true
    disable_ipv6 = true
    network_id   = var.network_id
  }]

  load_balancer_type                   = var.load_balancer_type
  load_balancer_location               = var.load_balancer_location
  load_balancer_disable_ipv6           = true
  load_balancer_disable_public_network = false

  enable_delete_protection = {
    floating_ip   = true
    load_balancer = true
    volume        = true
  }

  disable_hetzner_csi               = false
  hetzner_ccm_use_helm              = true
  allow_scheduling_on_control_plane = false

  system_upgrade_use_drain = true
  automatically_upgrade_os = true

  cluster_name                  = "${var.environment}-cluster"
  use_cluster_name_in_node_name = false

  firewall_kube_api_source = [
    var.management_network_cidr, # management network CIDR range
  ]

  firewall_ssh_source = [
    var.management_network_cidr, # management network CIDR range
  ]

  dns_servers = [
    "8.8.8.8",
    "8.8.4.4",
    "1.1.1.1",
  ]

  use_control_plane_lb                     = true
  control_plane_lb_enable_public_interface = false

  create_kubeconfig    = false
  create_kustomization = false
  export_values        = false

  providers = {
    hcloud = hcloud
  }
}
```

### Screenshots

<img width="1150" height="43" alt="Image" src="https://github.com/user-attachments/assets/d975d18a-b54f-4f81-ae94-bfebdc27c5e4" />

### Platform

macOS, Linux

--- a/locals.tf
+++ b/locals.tf
@@ -12,7 +12,7 @@ locals {
 
   ccm_version    = var.hetzner_ccm_version != null ? var.hetzner_ccm_version : data.github_release.hetzner_ccm[0].release_tag
   csi_version    = length(data.github_release.hetzner_csi) == 0 ? var.hetzner_csi_version : data.github_release.hetzner_csi[0].release_tag
-  kured_version  = var.kured_version != null ? var.kured_version : data.github_release.kured[0].release_tag
+  kured_version  = length(data.github_release.kured) == 0 ? var.kured_version : data.github_release.kured[0].release_tag
   calico_version = length(data.github_release.calico) == 0 ? var.calico_version : data.github_release.calico[0].release_tag
 
   # Determine kured YAML suffix based on version (>= 1.20.0 uses -combined.yaml, < 1.20.0 uses -dockerhub.yaml)

